Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM node:7
+
+RUN npm install -g \
+      yarn \
+      ember-cli \
+      bower \
+ && apt-get update \
+ && apt-get install -y --no-install-recommends \
+      nginx \
+ && rm -rf /var/lib/apt/lists/* \
+ && rm /var/www/html/index.nginx-debian.html \
+ && ln -sf /dev/stdout /var/log/nginx/access.log \
+ && ln -sf /dev/stderr /var/log/nginx/error.log
+
+COPY . /usr/src/app
+
+WORKDIR /usr/src/app
+
+RUN yarn install \
```

Comment:
I think #98 changes this again that yarn is not used anymore... may we need to wait for that PR to be merged.

Reply:
Yeah, I missed that one and will rebase.

Reply:
I just pushed the rebase (and also used amend since I had to rebase anyway)